integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/spring/IntegrasjonspunktLocalPropertyEnvironmentPostProcessor.java

```diff
@@ -24,9 +23,13 @@
 public class IntegrasjonspunktLocalPropertyEnvironmentPostProcessor implements EnvironmentPostProcessor, ApplicationListener<ApplicationEnvironmentPreparedEvent> {
 
     private static final org.slf4j.Logger log = LoggerFactory.getLogger(IntegrasjonspunktLocalPropertyEnvironmentPostProcessor.class);
-    
+
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
+        if (Optional.ofNullable(environment.getProperty("app.local.properties.enable", Boolean.class)).orElse(true).booleanValue() == false) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1125)